ruby/tzdump.rb

```diff
@@ -5,35 +7,50 @@
 START_UTC = DateTime.new(1905, 1, 1, 0, 0, 0, '+0')
 END_UTC = DateTime.new(2035, 1, 1, 0, 0, 0, '+0')
 
+def puts_crlf(*messages)
+  $stdout.write(messages.join("\r\n"))
+  $stdout.write("\r\n")
+end
```

Comment:
As noted on previous comments, this doesn't work on Windows - we end up with "\r\r\n" between each line. We definitely need something like the earlier code... it's the only thing I've seen so far that does the right thing on Windows.